components/PageButtons/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import "../index.css";
+import "../stories.js";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
```

Comment:
`import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";` is missing.